proposals/0273-webengine-projection-mode.md

```diff
@@ -82,19 +81,15 @@ Generally, the window capabilities should refer to available text and image fiel
 
 If the `WEB_VIEW` template is currently active, the window capabilities change as the application is presenting content using the web document.
 
-The following text fields `mainField1`, `mainField2`, `mainField3`, `mainField4`, `statusBar`, `mediaClock` and `mediaTrack` should be used practically on all the base templates. 
-It is required that `WindowCapability.textFields` should not contain these text field names if the OEM has implemented the `WEB_VIEW` template without these text fields.
-The text fields `menuName` and `templateTitle` should be included in the capabilities if these fields are visible on the HMI.
-

```

Comment:
Can you please advise which agreed upon revision required the removal of this content?

Reply:
@theresalech The 26 one. The next chapter:
```
26. Author to update
The WEB_VIEW template is not allowed to to support textfields, 
softbuttons, graphics (inside or outside of the webview).
```

to

```
The WEB_VIEW template must not declare support in its WindowCapabilities for any softButtonCapabilities, the TextFields with name: 
mainField1-4, mediaClock, mediaTrack, nor the ImageFields with name: softButtonImage, menuIcon, graphic, or secondaryGraphic.
```
As we have added the chapter above , the removed chapter became redundant

Reply:
The concerning text is redundant to the text we agreed for 26 and doesn't include additional action items for this proposal.

> The following text fields `mainField1`, `mainField2`, `mainField3`, `mainField4`, `statusBar`, `mediaClock` and `mediaTrack` should be used practically on all the base templates.

This text is rhetorical and unrelated to the proposed feature. It's related to the following text:

> - It is required that `WindowCapability.textFields` should not contain these text field names if the OEM has implemented the `WEB_VIEW` template without these text fields.

Two things. First the accepted text from 26 says basically the same that text fields shouldn't be included. Second `if the OEM has implemented the WEB_VIEW template without` is contradicting to `must not declare support`. Therefore this paragraph is invalid as we accepted 26.

> - The text fields `menuName` and `templateTitle` should be included in the capabilities if these fields are visible on the HMI.

We agreed that the app should use `ShowAppMenu` RPC and render an own menu button hence `menuName` shouldn't be supported. Also, please note following text within the proposal `The HMI should include an app title and a template title.` which makes the concerning sentence also redundant.

Reply:
Thank you for the explanations!